### PR TITLE
[external-modals] open color-picker and line-spacing modals in innerM…

### PR DESCRIPTION
**Pull request by noamtwix**
…odal instead of react modal

### CODE REVIEW

File: packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const createToolbar: CreatePluginToolbar = config => {
   };
 
   const save = spacing => {
-    helpers?.closeModal?.();
+    isMobile ? helpers?.closeModal?.() : innerModal?.closeInnerModal?.();
```
Comment (shaulgo) [inline packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx:42]:
```suggestion
    closeModal()
```

File: packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx [hunk 2]
```diff
@@ -119,13 +114,22 @@ const createToolbar: CreatePluginToolbar = config => {
         fullScreen: false,
         isMobile,
       });
-      helpers?.openModal?.({
+      const modalProps = {
```
Comment (shaulgo) [inline packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx:117]: const openModal = isMobile ? helpers.openModal : innerModal.openInnerModal;
openModal(modalProps)

File: packages/plugin-text-color/web/src/toolbar/getTextColorButtonProps.js [hunk 3]
```diff
@@ -56,7 +57,7 @@ export const getButtonProps = ({ config, type }) => {
     const editorState = getEditorState();
     const selection = editorState.getSelection();
     setEditorState(EditorState.forceSelection(newEditorState || editorState, selection));
-    helpers?.closeModal?.();
+    isMobile ? helpers?.closeModal?.() : innerModal?.closeInnerModal?.();
```
Comment (shaulgo) [inline packages/plugin-text-color/web/src/toolbar/getTextColorButtonProps.js:60]: const closeModal = isMobile ?...
closeModal()

File: packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ const spaceAfter = 'padding-bottom';
 
 //TODO: refactor code duplication here and in the LineSpacingButton
 const createToolbar: CreatePluginToolbar = config => {
```
Comment (shaulgo) [inline packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx:23]: add changelog

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx (hunk 4 above)
Comment (noamtwix) [reply packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx:23]: @shaulgo 
it is under `bug fix` or `internal`?

Comment (lxgreen) [reply packages/plugin-line-spacing/web/src/toolbar/createToolbar.tsx:23]: this should go under 'adding a new bug'. this change is going to screw an external toolbar positioned outside the editor container

**Pull request closed without merging**